src/DynamoCoreWpf/Properties/Resources.resx

```diff
@@ -2537,15 +2537,15 @@ Uninstall the following packages: {0}?</value>
     <comment>Label used in the general tab</comment>
   </data>
   <data name="PreferencesNodeAutocompleteMethod" xml:space="preserve">
-    <value>Default method</value>
+    <value>Default Ranking Method</value>
     <comment>Label used in the features tab</comment>
   </data>
   <data name="ObjectType" xml:space="preserve">
-    <value>Object type</value>
+    <value>Node Type Match</value>
```

Comment:
Is this changed confirmed from Sol or the UX team. I saw a comment from Helena regarding this change but not sure if it was finalized.

Reply:
> Is this change confirmed from Sol or the UX team. I saw a comment from Helena regarding this change but not sure if it was finalized.

just confirmed by @Helena

Reply:
Is the terminology also changed in preferences panel?

Reply:
Thinking about this more, in the future we are also going to show nodes based on usage. This will be under the same section where we show ML recommended nodes. So we should have a title that suits both the cases.

Reply:
@Jingyi-Wen thoughts?

Reply:
> Is the terminology also changed in preferences panel? I mean, does this change reflect in the context menu of node-autocomplete window and also the preferences panel.

yes in the preference panel, actually the change is in the resources files so it reflects also in the node autocomplete context menu automatically